### ISSUE TITLE
So...... How exactly do I create an alias?

### ISSUE THREAD

**Issue by Peter-Schorn** [al45tair/mac_alias#5]
All I want to do is one very simple thing: I want to  create an alias for a file. I would like to pass the POSIX path of the source file and the POSIX path of the destination file as variables into an argument. Here's an example of what I want to do written in plain english.

Make alias of (POSIX path of source file) and place the alias in (POSIX path of destination file)

Now, how do I convert this into python code? I read through the documentation, but, unfortunately, it didn't help me :( 

I would really appreciate the help, as I'm extremely new to coding.

EDIT: I did some more digging and, based on what I've read, mac_alias does not create aliases for macOS finder items. Does anyone know of a way of doing this via python?

**Comment by al45tair** [comment]
I don't think this module is really what you're looking for; it sounds like you probably want to make a symlink using [the os.symlink function](https://docs.python.org/3.8/library/os.html#os.symlink).

The mac_alias module is for making Alias and Bookmark _data structures_, not alias files. You _could_ use it to create an alias file, with a bit of extra work, since they're some kind of encapsulated Bookmark these days, but it isn't something I've bothered with.
